pytheus/backends.py

```diff
@@ -4,15 +4,14 @@
 
 from abc import ABC, abstractmethod
 from threading import Lock
-from typing import Any, TYPE_CHECKING
-
-import redis
+from typing import Any, Type, TYPE_CHECKING
 
 from pytheus.exceptions import InvalidBackendClassException, InvalidBackendConfigException
 
 
 if TYPE_CHECKING:
     from pytheus.metrics import Metric
+    import redis
```

Comment:
This is a change to make this work without having `redis` installed, unless you are using the Redis backend of course.

Reply:
Essentially all the changes in this file are to pass mypy, I caught a few issues with it :slightly_smiling_face: